_posts/2022-09-03-aphp-user-story.adoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: 'Quarkus adoption by APHP (Assistance Publique des Hôpitaux de Paris)'
+date: 2022-09-05
```

Comment:
Can you adjust the date to next Tuesday. Same for the filename?

If you can get the suggestions in by next Tuesday, I will publish it and post about it on our Twitter account.